tests/core/test_channels.py

```diff
@@ -482,14 +482,16 @@ def test_slack_metadata():
     user = "user1"
```

Comment:
`from sanic.request import Request` (l.480) is unused in this test, so can be removed